### ISSUE TITLE
Spike - Dev Research Inline Images to Full Statement/Notes

### ISSUE THREAD

**Issue by Vishvesh-LM** [opensalt/opensalt#530]
We need to research the ability to add inline images to Full Statement/Notes in either Add or Edit modes before we can accurately point the story for development work.

**Comment by Vishvesh-LM** [comment]
Below are the details of the story that is dependent upon this spike task:

**Acceptance Criteria:**

GIVEN an OS Editor
WHEN they add a new framework or item
THEN the toolbar contains a button to allow the addition of an embedded image

GIVEN an OS Editor
WHEN they edit a framework or item
THEN the toolbar contains a button to allow the addition of an embedded image

GIVEN an OS Editor
WHEN they click on the 'add inline image' button in the toolbar in either 'add' or 'edit' mode for Full Statement or Notes fields
THEN the image is embedded in the Full Statement or Notes and renders properly

GIVEN a Framework with inline images
WHEN using the 'Copy Framework' button
THEN the inline images are retained in the new Framework and render properly

GIVEN a Framework with inline images
WHEN using the 'Export' button
THEN the inline images are exported and render properly

**Comment by Vishvesh-LM** [comment]
Below is the Description of the story that is dependent on the spike task:

As an OS editor
I want to be able to select a button to insert an inline image  
So that I can add images to the Full Statement and Notes fields in 'Add' or 'Edit' modes

SCOPE
The scope of this story is to allow users to click on a button in the toolbar in order to add embedded images to Full Statement or Notes fields.  Per Barb's note - Looks like code mirror doesn't allow alt text

OUT OF SCOPE
Adding inline images to the Comments section is out of scope at this time.  Attachments story may take care of this (KMS-3929)

Adding inline images via 'Import Children' is out of scope at this time

AUTOMATED ACCEPTANCE TEST CASES
< need input from team>

DOMAIN
Login to OS Staging environment:  [http://frameworks-staging.act.org/login]

OS > Select Framework > Select Parent or Child Item > 'Edit' button > Full Statement AND Notes text box
OS > Select Framework > Select 'Add New Child Item' button > Full Statement AND Notes text box
OS > Select Framework > Select 'Export' button 
OS > Select Framework > Select 'Copy Framework' button
 
DOCUMENTATION

https://opensalt.mybalsamiq.com/projects/opensalt/CF%20Item%20Edit

**Comment by brandonopened** [comment]
This spike closed by #541
